### ISSUE TITLE
batch run

### ISSUE THREAD

**Issue by mabouelk** [ncbi/amr#32]
I wanna run amr on multiple files at once (Batch run) using protein sequence, nucleotide sequence and gff sequence. 

I tried that bu tit did not work 

amrfinder -p $sample.faa -g $sample.gff -n $sample.fsa but it did not work

**Comment by vbrover** [comment]
We run `amrfinder` in parallel on multiple files routinely.
Does `amrfinder` work for you on one file?

**Comment by mabouelk** [comment]
yes it works for me on one file.

amrfinder -p sample.gff -g sample.gff -n sample.fsa --O Staphylococcus_aureus --plus 

can you provide me the command for doing the same on multiple files as I have more than 200 samples? 

also I have gff output from prokka and I haave to run this on each file to fix it 
perl -pe '/^##FASTA/ && exit; s/(\W)Name=/$1OldName=/i; s/ID=([^;]+)/ID=$1;Name=$1/' <prokka_output.gff>  > <amrfinder.gff>

can you provide me the code for that to fix all gff file at once

**Comment by mabouelk** [comment]
thank you so much the last one worked after I split it into two commands one for fixing .gff files and the other for amfinder. You are amazing!
